src/globus_cli/commands/logout.py

```diff
@@ -142,7 +142,7 @@ def logout_command(*, login_manager: LoginManager, ignore_errors):
 
         adapter.remove_tokens_for_resource_server(rs)
 
-    adapter.remove_config(_STORE_CONFIG_USERINFO)
+    remove_well_known_config("auth_user_data")
```

Comment:
`remove_well_known_config` is basically the same as the above line, but it relies on `Literal[...]` type-safety to ensure we always target the correct strings.

Therefore, when I made the change which introduced it, and the other helpers like it, I made sure to do a quick pass over the annotations for the modules where they were used to be certain that `mypy` was type-checking there. That's why each place you see one of these `*_well_known_config` helpers used, you will likely see other changes to annotations.